software/lazydogpile.livecodescript

```diff
@@ -0,0 +1,954 @@
+   put item 4 of sNodeDetailsGeometry["tagDataList"] into tOldFloor
+   put pHeight-tOldFloor into tHeightDelta
+   put pWidth/2 into tHalfWidth
+   put tHalfWidth-tOldMid into tWidthDelta
+   put pWidth-5 into tPadWidth
+   put pHeight-5 into tPadHeight
+   
+   put tHalfWidth into item 3 of sNodeDetailsGeometry["tagDataList"]
+   put tPadHeight into item 4 of sNodeDetailsGeometry["tagDataList"]
+   put tHalfWidth+5 into item 1 of sNodeDetailsGeometry["tagLabel"]
+   put tPadWidth into item 3 of sNodeDetailsGeometry["tagLabel"]
+   put tHalfWidth+5 into item 1 of sNodeDetailsGeometry["editData"]
+   put tPadWidth into item 3 of sNodeDetailsGeometry["editData"]
+   put tPadHeight into item 4 of sNodeDetailsGeometry["editData"]
+   put item 1 of sNodeDetailsGeometry["loadImgButt"] into tX
+   put tX+tWidthDelta into item 1 of sNodeDetailsGeometry["loadImgButt"]
+   put item 1 of sNodeDetailsGeometry["clearImgButt"] into tX
+   put tX+tWidthDelta into item 1 of sNodeDetailsGeometry["clearImgButt"]
+   put item 1 of sNodeDetailsGeometry["expandImgButt"] into tX
+   put tX+tWidthDelta into item 1 of sNodeDetailsGeometry["expandImgButt"]
+   put item 1 of sNodeDetailsGeometry["setCaptureButt"] into tX
+   put tX+tWidthDelta into item 1 of sNodeDetailsGeometry["setCaptureButt"]
+   put item 1 of sNodeDetailsGeometry["snapCaptureButt"] into tX
+   put tX+tWidthDelta into item 1 of sNodeDetailsGeometry["snapCaptureButt"]
+   put pWidth into sNodeDetailsGeometry["startWidth"]
+   put pHeight into sNodeDetailsGeometry["startHeight"]
+   
+   set the rect of field "tagDataList" stack "NodeDetails" to sNodeDetailsGeometry["tagDataList"]
+   set the rect of field "tagLabel" stack "NodeDetails" to sNodeDetailsGeometry["tagLabel"]
+   set the rect of field "editData" stack "NodeDetails" to sNodeDetailsGeometry["editData"]
+   set the loc of button "loadImgButt" stack "NodeDetails" to sNodeDetailsGeometry["loadImgButt"]
+   set the loc of button "clearImgButt" stack "NodeDetails" to sNodeDetailsGeometry["clearImgButt"]
+   set the loc of button "expandImgButt" stack "NodeDetails" to sNodeDetailsGeometry["expandImgButt"]
+   set the loc of button "setCaptureButt" stack "NodeDetails" to sNodeDetailsGeometry["setCaptureButt"]
+   set the loc of button "snapCaptureButt" stack "NodeDetails" to sNodeDetailsGeometry["snapCaptureButt"]
+   
+   get the filename of image "figureHolder" stack "NodeDetails"
+   if it is not empty then 
+      put the rect of field "editData" stack "NodeDetails" into tQuery["maxRect"]
+      put the long id of image "figureHolder" stack "NodeDetails" into tQuery["longID"]
+      put fitImageToConstraint(tQuery) into tGimme
+      set the width of image "figureHolder" stack "NodeDetails" to tGimme["width"]
+      set the height of image "figureHolder" stack "NodeDetails" to tGimme["height"]
+      set the loc of image "figureHolder" stack "NodeDetails" to tGimme["center"]
+   end if
+end resizeNodeDetails
+
+on expandImgButtMouseUp
```

Comment:
Yeah, that was a way to move the script out of the button to the script-only stacks. It looks like you're saying it would be better to set the behavior of a button from a script-only stack instead?